nad_ch/application/data_reader.py

```diff
@@ -0,0 +1,62 @@
+from geopandas import GeoDataFrame, read_file
+import fiona
+from typing import Optional
+import yaml
+import os
+
+
+class DataReader(object):
+    def __init__(self, config_name: Optional[str] = None) -> None:
+        self.config_name = config_name
+        self.default_config_path = "nad_ch/application/nad_column_maps/default.yaml"
```

Comment:
This approach seems reasonable to me in the context of an MVP, but we might want to keep in mind the option of   having the Storage infrastructure class handle configuration file storage and retrieval at the behest of the application layer, as opposed to having the DataReader class know about the filesystem.

That being said, let's roll with whatever is going to get us up and running quickly and keep an eye on it, imo. @danielnaab please chime in if you have other thoughts.